teamA/jinwoo/models/postDao.js

```diff
@@ -0,0 +1,81 @@
+const { DataSource } = require('typeorm');
+
+const mysqlDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+mysqlDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!")
+  })
+  .catch((err) => {
+    console.error("Error during Data Source initialization", err);
+      mysqlDataSource.destroy();
+  });
+
+  const createPost = async ( title, content, userId ) => {
+    try {
+      return await mysqlDataSource.query(
+        `INSERT INTO postsWesta(
+          title,
+          content,
+          user_id
+        ) VALUES (?, ?, ?);
+        `,
+        [ title, content, userId ]
+      );
+    } catch (err) {
+      const error = new Error('INVALID_DATA_INPUT');
+      error.statusCode = 500;
+      throw error;
+    }
+  };
+
+  const showPosts = async () => {
+      return await mysqlDataSource.query(
+        `SELECT
+            usersWesta.id as userId,
+            usersWesta.profile_image as userProfileImage,
+            postsWesta.id as postingId,
+            postsWesta.title as postingTitle,
+            postsWesta.content as postingContent
+          FROM usersWesta
+          INNER JOIN postsWesta
+          ON postsWesta.user_id = usersWesta.id
+          `
+        );
+    };
+
+  const delPosts = async ( postId ) => {
```

Comment:
```suggestion
  const deletePost = async ( postId ) => {
```
축약어 보다는 본래 단어를, 해당 쿼리는 하나의 포스트를 지우는 쿼리이기에 단수로 네이밍을 해주세요 :)